src/Illuminate/Foundation/Console/stubs/rule.invokable.implicit.stub

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace {{ namespace }};
+
+use Illuminate\Contracts\Validation\InvokableRule;
+
+class {{ class }} implements InvokableRule
+{
+    /**
+     * Indicates whether the rule should be implicit.
+     *
+     * @var bool|null
```

Comment:
Although I understand why you've put `null` in the type, if this property exists, I don't think it makes sense to indicate that it is `null`. It should only be `true` or `false` if the property exists.

Yes, `null` is supported, but it isn't intended to be set to `null`.

```suggestion
     * @var bool
```

Reply:
I agree with you
Actually what I had in mind is that : "this property is not required", I was trying to express that :)